### PR TITLE
Jordan/cleanup and about page

### LINKED ISSUE #1863: Add a footer
We need to show certain information when Voyager is a web page:

...

### LINKED ISSUE #2353: Scrolling in Firefox on Mac on PageValidator is broken
**Describe the bug**

Apparently the scrolling is broken there according to one user.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by jbibla**
closes #2353 
closes #1863

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
Screen Shot 2019-04-06 at 1 35 19 PM
Screen Shot 2019-04-06 at 1 35 35 PM
Screen Shot 2019-04-06 at 1 37 35 PM
Screen Shot 2019-04-06 at 1 38 08 PM

File: app/src/renderer/components/common/PageAbout.vue [hunk 1]
```diff
@@ -1,53 +1,63 @@
...
-          from validators so
-          there is an entity you can trust.
+          Lunie is your one-stop-shop for sending and receiving Cosmos tokens,
+          viewing your transaction history, delegating your ATOMs, and
+          participating in governance on the Cosmos Hub.
+        </p>
+        <p>
+          This software is developed by the team at Lunie Internation Software
+          Systems. It's
+          <a href="https://github.com/luniehq/lunie" target="_blank">
```
Comment (fedekunze) [inline app/src/renderer/components/common/PageAbout.vue]:
```suggestion
          <a href="https://github.com/luniehq/lunie" target="_blank" rel="noopener noreferrer">
```

File: app/src/renderer/components/common/PageAbout.vue [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
+            open source
+          </a>
+          and built for the
+          <a href="https://cosmos.network" target="_blank">Cosmos</a> community.
```
Comment (fedekunze) [inline app/src/renderer/components/common/PageAbout.vue]:
```suggestion
          <a href="https://cosmos.network" target="_blank" rel="noopener noreferrer">Cosmos</a> community.
```

File: app/src/renderer/components/common/PageAbout.vue [hunk 3]
```diff
@@ -1,53 +1,63 @@
...
             <img src="~assets/images/team/fabo.jpg">
-            <span>Fabian Weber<br>CTO<br>Germany</span>
+            <div class="profile-info">
+              <span>Fabian Weber</span>
+              <span class="flag">🇩🇪</span>
+            </div>
           </div>
           <div class="profile">
-            <img src="~assets/images/team/fede.jpg">
-            <span>Federico Kunze<br>Software Engineer<br>Chile</span>
```
Comment (fedekunze) [inline app/src/renderer/components/common/PageAbout.vue]: 😢

File: app/src/renderer/components/common/PageAbout.vue [hunk 4]
```diff
@@ -1,53 +1,63 @@
...
       <div class="card">
         <h1>Contact</h1>
         <p>
-          We are happy about any feedback or question. Write us at:<br>
-          <br>
-          <a href="mailto:contact@lunie.io">contact[at]lunie.io</a><br>
-          <a href="https://github.com/cosmos/lunie">GitHub</a><br>
-          <a href="https://forum.cosmos.network/c/wallet">Cosmos Forum</a>
+          Please let us know what you think! You can find us on
+          <a href="https://github.com/cosmos/lunie" target="_blank">GitHub</a>,
```
Comment (fedekunze) [inline app/src/renderer/components/common/PageAbout.vue]: ditto

File: app/src/renderer/components/common/PageAbout.vue [hunk 5]
```diff
@@ -1,53 +1,63 @@
...
+          email us at
+          <a href="mailto:contact@lunie.io" target="_blank">
```
Comment (fedekunze) [inline app/src/renderer/components/common/PageAbout.vue]: ditto

File: app/src/renderer/components/common/PageAbout.vue [hunk 6]
```diff
@@ -1,53 +1,63 @@
...
+          <a href="https://cosmos.network" target="_blank">Cosmos</a> community.
+        </p>
+        <p>
+          For your safety and security, Lunie will not ask for or store your
+          private keys or seed phrases. For now, to send transactions with
+          Lunie, you'll have to sign them with your Ledger Nano&nbsp;S.
+        </p>
+        <p>
+          You can try Lunie on our testnet by visiting
+          <a href="https://testnet.lunie.io" target="_blank">
```
Comment (fedekunze) [inline app/src/renderer/components/common/PageAbout.vue]: ditto

**Pull request merged**